### ISSUE TITLE
Broken FavIcon

### ISSUE THREAD

**Issue by benyaminl** [carlosdoub/tab_list_button_enhanced#2]
![image](https://user-images.githubusercontent.com/2817332/123419874-f7b49980-d5e4-11eb-9160-f4f1c3647787.png)
Broken favIcon if it's local address, need to be changed with other Icon I think

Example :
reader mode, addons page, settings page, about config, etc

**Comment by carlosdoub** [comment]
Yes, I know. In the code there are hotfixes for the case of about:addons URL, for example. If not, the loading icons would remain spinning.
It would be good to use for example a generic icon for about pages, or something like that.

**Comment by benyaminl** [comment]
> 
> 
> Yes, I know. In the code there are hotfixes for the case of about:addons URL, for example. If not, the loading icons would remain spinning.
> It would be good to use for example a generic icon for about pages, or something like that.

I will implement it, is it okay, as I already skim your code 

It's realllyyy understandable code. I really thankful for it

**Comment by carlosdoub** [comment]
Great! go ahead

**Comment by benyaminl** [comment]
Related to #4 , For internal FavIcon

**Comment by benyaminl** [comment]
@carlosdoub if you has time, probably please test also this, as I fix this problem using lastest code, I think it already landed on AMO? hmm.... because I haven't seen any broken favicon, and no favicon replaced with firefox brand icon

**Comment by carlosdoub** [comment]
If you watch the commit history of main, I've added your previous commits, plus some changes I've made for the reloading functionality 

But the about:addons URL still doesn't work, can't even replace it with Firefox icon. The firefox icon works great on about:blank, which I use a lot.

You tell me I should integrate the current open PR? I would be ok, I see there are some changes I haven't added.

**Comment by benyaminl** [comment]
![image](https://user-images.githubusercontent.com/2817332/124524943-61c01080-de27-11eb-9496-87e29e480526.png)
Hmm... I have add if there for mozapp folder

https://github.com/benyaminl/tab_list_button_enhanced/commit/4c7b8e4df8ca02f3ca4d1319ea84f8425f6da7a8

hmmmm

EDIT: OH it's not on main branch yet, sorry, it's still on pull request, pardon me

**Comment by carlosdoub** [comment]
Hello, I've messed up the commits, so had to reset the repository to a previous commit. You can continue colaborating if you want. The feature you mention in the previous comment I found it not much better than was already in the addon, so far.

**Comment by benyaminl** [comment]
> 
> 
> Hello, I've messed up the commits, so had to reset the repository to a previous commit. You can continue colaborating if you want. The feature you mention in the previous comment I found it not much better than was already in the addon, so far.

For the favicon? At least it doesn't break for now, but I think we must put default icon on addons and eliminate at all chrome:// icon with our icon, how about it?

EDIT : Yeah this still happen, I think we need to replace any moz-extension, or chrome:// page to our favicon

![image](https://user-images.githubusercontent.com/2817332/124853971-eb5f1200-dfd0-11eb-9ae4-584515a340ee.png)
That's with singlepage html saver extension

**Comment by carlosdoub** [comment]
For some reason the icon in about:addons wasn't showing after a reload, even if it shows at first load. That is good anyway, but yes, we could search a favicon
But what I don't found better that was already, was the fix when less than 5 tabs. That is what I meant.

**Comment by benyaminl** [comment]
> 
> 
> For some reason the icon in about:addons wasn't showing after a reload, even if it shows at first load. That is good anyway, but yes, we could search a favicon
> But what I don't found better that was already, was the fix when less than 5 tabs. That is what I meant.

Yeah next I think I will rollback those, I think about implementhing CSS scrollbar rather than System default scroll so we can save space and fix the width at the same time. 

https://stackoverflow.com/questions/16670931/hide-scroll-bar-but-while-still-being-able-to-scroll/49278385#49278385 

I don't know would I work, but hopefully, I will come back after I done all my exam. Thanks
